### PR TITLE
Fixed bug: Dino does not add own account to roster

### PR DESCRIPTION
**Pull request by Psayker**

### CODE REVIEW

File: main/src/ui/add_conversation/roster_list.vala [hunk 1]
```diff
@@ -34,7 +34,13 @@ protected class RosterList {
             foreach (ulong handler_id in handler_ids) stream_interactor.get_module(RosterManager.IDENTITY).disconnect(handler_id);
         });
 
-        foreach (Account a in accounts) fetch_roster_items(a);
+        foreach (Account a in accounts){
```
Comment (mxlgv) [inline main/src/ui/add_conversation/roster_list.vala:37]: Please fix codestyle: `foreach (...) {`

File: main/src/ui/add_conversation/roster_list.vala [hunk 2]
```diff
@@ -34,7 +34,13 @@ protected class RosterList {
...
+            ListRow own_account_row = new ListRow.from_jid(stream_interactor, a.bare_jid, a, accounts.size > 1);
+            ListBoxRow own_account_lbrow = new ListBoxRow() { child=own_account_row };
```
Comment (mxlgv) [inline main/src/ui/add_conversation/roster_list.vala:39]: Please fix codestyle: `child = own_account_row`

**Review by mxlgv — CHANGES_REQUESTED** [review]

File: main/src/ui/add_conversation/roster_list.vala (hunk 1 above)
Comment (mxlgv) [reply main/src/ui/add_conversation/roster_list.vala]: @Psayker now `(` is stuck to `foreach`

Comment (Psayker) [reply main/src/ui/add_conversation/roster_list.vala]: done

**Review by mxlgv — APPROVED** [review]

**Pull request merged**